plugins/imfile/imfile.c

```diff
@@ -1295,6 +1310,39 @@ getStateFileName(const act_obj_t *const act,
 	return buf;
 }
 
+static rsRetVal
+checkPerMinuteRateLimits(per_minute_rate_limit_t *per_minute_rate_limits,
+			     const size_t msgLen)
+{
+	DEFiRet;
+	time_t current_minute = time(NULL)/60;
```

Comment:
minutes since the epoch. Safer than current minute of the day as this way each minute will only ever occur once.